frontend/src/api/index.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const API_URL = 'http://localhost:4000';
+
```

Comment:
This can be improved:

```
const options = {
  baseURL: 'http://localhost:4000' // process.env.REACT_APP_BACKEND_API
};

const myAxios = axios.create(options);
myAxios.defaults.timeout = 2500;

const { CancelToken } = myAxios;

let cancel;

const cancellation = () => ({
  cancelToken: new CancelToken(function executor(canceller) {
    cancel = canceller;
  })
});

get:(url) => {
   return myAxios.get(`url`, cancellation); // example: url = 'stores' 
},
```